nservicebus/scalability-and-ha/scale-out.md

```diff
@@ -60,16 +60,16 @@ WARNING: When using this scaling out technique in a mixed version environment ma
 
 ### Broker transports
 
-The main difference when using broker transports is that queues are not attached to machines but are rather maintained by a central server (or cluster of servers). When running on a broker transport such as RabbitMQ or SQL Server, it is enough to specify the logical routing:
+The main difference when using broker transports is that queues are not attached to machines but rather are maintained by a central server (or cluster of servers). Such design enables usage of the Competing Consumers pattern for scaling out processing power. All instances have empty IDs and connect to the same infrastructure queue. When running on a broker transport such as RabbitMQ or SQL Server, it is enough to specify the logical routing:
```

Comment:
It is relevant for both but I decided to explicitly split this article into V5 and V6 parts. The usage of competing consumers pattern is described in the V5 part in both SQL/Rabbit and Azure sections.